### DISCUSSION TITLE
Meeting: October 1st, 2024

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#155] [Announcements]
## Weekly meetings happen on Tuesdays

**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

### Accessibility & Etiquette

* Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).
* Meetings are recorded for future reference, and for those who are not able to attend in-person.
* We invited challenges to language or behaviour that is harmful or not inclusive during this meeting.
* We look forward to your participation, but please consider these acts of etiquette:
  * Remain on mute when not speaking to prevent interruptions.
  * Blur your background to reduce visual distractions.
  * Use the Zoom meeting "Raise Hand" feature to notify the group when you wish to speak.

| Blur My Background | Raise Hand |
|-|-|
| <img width="323" alt="Screenshot of Zoom UI showing the 'Stop Video' and 'Blur My Background' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/7e43dbbb-6529-46e6-8b04-4c1aa852d9dd"> | <img width="323" alt="Screenshot of Zoom UI showing the 'Reaction' and 'Raise Hand' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/f991722f-4651-40aa-9bc4-7e9a2a165a6a"> |

### Agenda Structure

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Wrap-up and seed the next agenda (5 mins) | SIG members | -- |

Whether attending or not, **anyone can comment on this issue prior to the meeting to suggest topics or to add comments on planned topics or proposals**.

**Comment by earth2marsh** [comment]
For this week we intend to discuss the orange group in the diagram from @handrews 's recent [blog post](https://modern-json-schema.com/analyzing-the-openapi-tooling-ecosystem): 
![image](https://github.com/user-attachments/assets/202c3da0-2d76-465f-83e1-ff3bb38600b7)

A question to seed the discussion: what are the primary structural elements that are part of the HTTP concerns? Then can we map the Moonwalk structure to those HTTP concerns?

**Comment by earth2marsh** [comment]
Today we began with a high level discussion of the idealized architecture diagram and debated some possible alternative labels for the three major divisions (in bright purple):
![2024 10 01 Moonwalk Architecture feedback](https://github.com/user-attachments/assets/fc5132bd-168e-4db0-896a-1f834435098e)

One other suggestion to consider is whether it makes sense to also represent the DOM in the section currently labeled "Parsing Libraries." 

We then discussed the orange boxes and wrestled with the fact that some of them (on the left) are more mechanically oriented while on the right are more semantically aligned. There was some discussion of how orange may overlap with blue (pick that up next week!).

Folks that attended or watched the [video](https://zoom.us/rec/share/ruoh0FwkhdUKwZ2JnzDxfdHVz02lowgNmNOkCZrrvIfslZFtneeDcqP88KgfIQFm.HX1W_OweQJAnnrC5) (added later!), please correct or add anything that I may have missed in the recap that you found pertinent!

**Reply by benny-yamagata** [reply]
I think `Authoring/Editing` or even `Creating/Editing` would be clearer than manipulating.
